rheem-platforms/rheem-flink/src/main/java/org/qcri/rheem/flink/channels/DataSetChannel.java

```diff
@@ -0,0 +1,100 @@
+package org.qcri.rheem.flink.channels;
+
+import org.apache.flink.api.common.functions.FilterFunction;
```

Comment:
Please remove unused imports, specially when the commit message is "Clean of the code" :)